frontend/integration-tests/tests/storage-dashboard.scenario.ts

```diff
@@ -0,0 +1,31 @@
+import { browser } from 'protractor';
+import { appHost } from '../protractor.conf';
+import { dashboardIsLoaded } from '../views/dashboard-shared.view';
+import { serviceName, clusterHealth, clusterName, allNodes } from '../views/storage-dashboard.view';
+import { execSync } from 'child_process';
+
+describe('Check data on Storage Dashboard.', () => {
+  beforeAll(async() => {
+    await browser.get(`${appHost}/dashboards/persistent-storage`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check cluster health is OK', async() => {
+    expect(clusterHealth.getText()).toContain('is healthy');
+  });
+
+  it('Check service name is OCS', async() => {
+    expect(serviceName.getText()).toEqual('OpenShift Container Storage');
+  });
+
+  it('Check that cluster name is correct', async() => {
+    const cephClusterName = execSync('oc describe cephcluster -n openshift-storage|sed -n "s/^Name: \\+//p"');
```

Comment:
Could you use `kubectl` instead of `oc`? And it's better to inspect JSON instead of describe. Describe is intended for users, not really for automation. 
```
const cephcluster = JSON.parse(
  execSync('kubectl get cephcluster -n openshift-storage -o json', { encoding: 'utf-8' })
);
expect(clusterName.getText()).toEqual(_.get(cephcluster, 'path.to.the.cephcluster.name', ''))
```

Reply:
Fixed